.github/workflows/acquia_cms_ci.workflow.yml

```diff
@@ -164,7 +164,6 @@ jobs:
           # Update the CI by adding patches without pinning the following modules.
           composer require "drupal/facets:^2.0.6" --no-update --no-install -d modules/acquia_cms_search
           composer require "drupal/pathauto:^1.11" --no-update --no-install -d modules/acquia_cms_common
-          composer require "drupal/redirect:^1.8" --no-update --no-install -d modules/acquia_cms_common
```

Comment:
I don't think this change is related to this ticket.

Reply:
This is cherry-picked from ACMS-1947, to make CI runs